<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormTienda</class>
 <widget class="QDialog" name="FormTienda">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox_Tienda">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>19</y>
     <width>351</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_nombre_tienda">
      <property name="text">
       <string>Nombre de la tienda:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_Nombre_tienda"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_Direccion_Internet_tienda">
      <property name="text">
       <string>Direccion de internet:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_Direccion_internet_tienda"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_Direccion_Fisica_tienda">
      <property name="text">
       <string>Direccion fisica:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_Direccion_Fisica_tienda"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_Telefono_tienda"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_Telefono_tienda">
      <property name="text">
       <string>Telefono:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox_Tienda</sender>
   <signal>accepted()</signal>
   <receiver>FormTienda</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox_Tienda</sender>
   <signal>rejected()</signal>
   <receiver>FormTienda</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
